travis-scripts/_FontForge

```diff
@@ -132,7 +132,7 @@ export PATH="$PATH:$bundle_bin"
 echo $WRAPPER $bundle_bin/fontforge > /tmp/oo
 
 if [ $# -eq 0 ] ; then
-    ( exec $WRAPPER $bundle_bin/fontforge -new & )
+    ( exec $WRAPPER $bundle_bin/fontforge -new  )
```

Comment:
This was so annoying, with this it actually uses the FontForge icon now, and it semi-works with opening files from Finder.

Reply:
Kinda makes sense. What would it mean to background a shell `exec` anyway ?

Reply:
It was probably a workaround for some other issue back when it was an X application, I vaguely recall mention of the dock icon continuously bouncing (this doesn't happen any more from what I can tell).